ngraph/core/include/ngraph/function.hpp

```diff
@@ -138,6 +144,24 @@ namespace ngraph
         bool evaluate(const HostTensorVector& output_tensors,
                       const HostTensorVector& input_tensors) const;
 
+        /// Return a list of function's sinks.
+        const SinkVector& get_sinks(void) const { return m_sinks; }
+        /// \brief Add new sink nodes to the list to avoid removing. 
```

Comment:
I guess it will be enough 
// \brief Add new sink nodes at the end of the list of sinks.

to avoid removing.  - looks odd. removing? from graph or not?

Reply:
fixed